src/components/ChannelPreview/ChannelPreviewMessenger.js

```diff
@@ -76,11 +77,9 @@ class ChannelPreviewMessenger extends PureComponent {
             <span>{displayTitle}</span>
           </div>
           <div className="str-chat__channel-preview-messenger--last-message">
-            {!channel.state.messages[0]
-              ? t('Nothing yet...')
-              : truncate(this.props.latestMessage, {
-                  length: this.props.latestMessageLength,
-                })}
+            {truncate(this.props.latestMessage, {
```

Comment:
It seems we don't have `lodash.truncate` listed as a [dependency](https://github.com/GetStream/stream-chat-react/blob/master/package.json#L32), though this [seems to be working](https://github.com/GetStream/stream-chat-react/pull/362/files#diff-b3b7e069465004ee21899b98498a8ec2R58). Probably worth taking the opportunity to add it explicitly, what you think?